parse/parse.go

```diff
@@ -483,3 +493,23 @@ func (t *Tree) parseVarQuote(start token) *VarQuoteNode {
 	}
 	panic("unreached")
 }
+
+func (t *Tree) includeNode(node Node) bool {
+	if list, ok := node.(*ListNode); ok {
```

Comment:
I think you can replace all this with

```
if t.ignoreCommentForm && FnFormSymbol(node, "comment") {
	return false
}
```

Reply:
Unfortunately that creates an import cycle since this code is in the parse package and that helper is defined in the goclj package. We could define a similar helper in the parse package and have goclj.FnFromSymbol call it to avoid breaking the public API, but just writing this one-off duplicate code seems ok too. WDYT?

Reply:
Sorry, I had missed that it's not in the main package. Leaving this as-is is fine.